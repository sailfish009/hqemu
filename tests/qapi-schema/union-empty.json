# FIXME - unions should not be empty
{ 'union': 'Union', 'data': { } }
