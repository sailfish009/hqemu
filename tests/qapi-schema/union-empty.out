object :empty
object Union
enum UnionKind []
